### ISSUE TITLE
explore android a11y

### ISSUE THREAD

**Issue by sashei** [mozilla-lockbox/lockbox-android#160]
this issue will constitutes an exploration of android a11y best practices. some potential resources include:
- other android teams at mozilla
- official android a11y documentation
- conference talks and blog posts about making accessible android apps

### acceptance criteria
- [ ] a document or email of findings to be shared with the rest of the team
- [ ] markdown writeup in `docs` about lockbox's a11y standards for future work
- [ ] follow-up issue to address any a11y debt extant in our currently-implemented user flows
- [ ] amendments to PR template to point contributors to relevant a11y resources when submitting new PRs

**Comment by sashei** [comment]
possible library resources via @m8ttyB :
- https://github.com/google/GTXiLib
- https://github.com/google/Accessibility-Test-Framework-for-Android

**Comment by eliserichards** [comment]
Info shared during the Android weekly team meeting (10/25)
- [Accessibility scanner](https://support.google.com/accessibility/android/answer/6376570?hl=en) automatically finds issues related to colorblindness, small text, small buttons, and generates reports
- [Introduction to programming for TalkBack](https://developer.android.com/guide/topics/ui/accessibility/apps) & others (screen reader)
- Don’t change focus programmatically
- “Try it as a user!”
- Developer options to simulate color blindness
- [Android Developers Backstage: Episode 14: Accessibility](http://androidbackstage.blogspot.com/2014/10/episode-14-accessibility.html)
- [Video introduction to accessibility](https://www.youtube.com/watch?v=1by5J7c5Vz4) and [Android Accessibility 101](https://youtu.be/UOr3mgqJU0A?list=PLnVy79PaFHMUqqvwbjyKJZv1N8rzHOCBi)
- :eeejay, Eitan, great for questions
- Automated testing is an unknown
-- Accessibility scanner to fix
-- [This doc?](https://developer.android.com/training/accessibility/testing#automated)
- Mike Comella to add to shared-docs

**Comment by sashei** [comment]
Shared document with that info, referenced above by @eliserichards : https://github.com/mozilla-mobile/shared-docs/pull/69 (currently still in PR form)

**Comment by mcomella** [comment]
Guide has landed: https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md 😄

**Comment by devinreams** [comment]
Once writeup doc #182 is complete, I'll "run and write-up" our test plan at #44 and aim to do that as part of this current sprint.
